packages/pluggable-widgets-tools/src/typings-generator/generateClientTypes.ts

```diff
@@ -224,15 +224,15 @@ export function toExpressionClientType(
         );
     }
 
-    const allowedTypes = ["Boolean", "DateTime", "Enum", "Integer", "Long", "String", "Decimal"];
-
-    if (!resolvedProperty.attributeTypes?.length || !resolvedProperty.attributeTypes[0]) {
+    const { attributeTypes } = resolvedProperty;
+    if (!attributeTypes?.length || typeof attributeTypes[0] === "string") {
```

Comment:
I was scratching my head a bit on that last one 😅. Maybe it would make more sense check for the presence of a `.attributeType` instead, since that is what we're using in the code below.

`if (!attributeTypes?.[0]?.attributeType) {` would be an equivalent check: `attributeTypes` needs to exist and it needs to have an element that has an `attributeType`.

Reply:
Oh, and `!= attribute` on line 221 should be `!==` :)